tests/test_point_range.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for Point.
+
+To run this unit test install UnitTesting package and choose
+"UnitTesting: Test current project" command.
+"""
+
+from unittest import TestCase
+
+from LSP.main import Point
+from LSP.main import Range
+
+
+class test_point(TestCase):
+
+    def test_valid_point(self):
+        lsp_point = {'line': 10, 'character': 4}
+        point = Point.from_lsp(lsp_point)
+        self.assertEqual(point.row, 10)
+        self.assertEqual(point.col, 4)
```

Comment:
Could test `to_lsp` at the same time
`self.assertEqual(lsp_point, Point.from_lsp(lsp_point).to_lsp())`